src/openrct2/actions/LargeSceneryRemoveAction.cpp

```diff
@@ -139,6 +142,9 @@ GameActions::Result::Ptr LargeSceneryRemoveAction::Execute() const
     tile_element_remove_banner_entry(tileElement);
 
     rct_scenery_entry* scenery_entry = tileElement->AsLargeScenery()->GetEntry();
+    // If we have a bugged scenery entry we cannot determine cost or size.
+    if (scenery_entry == nullptr)
+        return res;
```

Comment:
This is going to leave a broken tileelement. Perform this check prior to the tile_element_remove_banner_entry and also return a failure state. Not that this will mater as you can't get to here without first performing the query but best do things in the correct order.

Reply:
How is it going to _leave_ a broken tile element? If the tile is pointing to an unknown scenery entry, it is already broken. Might as well clean up what we can.

Reply:
But it doesn't do that. It just breaks it more. Just do the same make result here to match every other game action

Reply:
> But it doesn't do that. It just breaks it more.

Can you explain how?

Reply:
It removes the banner entry but not the element so you'll have a tile element that will have a bad banner element reference.

Reply:
I see.

I have made the requested changes.